osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -134,6 +137,68 @@ protected override void LoadComplete()
             updateState(ArmedState.Idle, true);
         }
 
+        private void apply(HitObject hitObject)
+        {
+#pragma warning disable 618 // can be removed 20200417
+            if (GetType().GetMethod(nameof(AddNested), BindingFlags.NonPublic | BindingFlags.Instance)?.DeclaringType != typeof(DrawableHitObject))
+                return;
+#pragma warning restore 618
+
+            if (nestedHitObjects.IsValueCreated)
+            {
+                nestedHitObjects.Value.Clear();
+                ClearNestedHitObjects();
+            }
+
+            foreach (var h in hitObject.NestedHitObjects)
+            {
+                var drawableNested = CreateNestedHitObject(h) ?? throw new InvalidOperationException($"{nameof(CreateNestedHitObject)} returned null for {h.GetType().ReadableName()}.");
+
+                drawableNested.OnNewResult += (d, r) => OnNewResult?.Invoke(d, r);
```

Comment:
should probably move this out into a separate method (duplicated code with `AddNested` below).